library/common/config/config.cc

```diff
@@ -252,83 +252,103 @@ const char* config_template = R"(
             budget_percent:
               value: 100
             min_retry_concurrency: 0xffffffff # uint32 max
+    # Used to reap dead connections. In mobile devices it is less important to keep a "host" ejected
```

Comment:
Follow up nit: this comment gets compiled in and just wastes binary space. I would probably move the comment and/or just move this config stanza out also and then use YAML include to include it.

Reply:
Here's a take on what that could look like:
https://github.com/envoyproxy/envoy-mobile/pull/1722